ooniprobe/View/TestResults/Rows/DashboardTableViewCell.m

```diff
@@ -45,7 +45,12 @@ -(void)setTestSuite:(AbstractSuite*)testSuite{
     
     [self.titleLabel setText:[LocalizationUtility getNameForTest:testSuite.name]];
     [self.descLabel setText:[LocalizationUtility getDescriptionForTest:testSuite.name]];
-    NSString *time = NSLocalizedFormatString(@"Dashboard.Card.Seconds", [NSString stringWithFormat:@"%d", [testSuite getRuntime]]);
+    //TODO convert seconds to minutes and hours when needed
+    //if getRuntime = 0 show one hour
+    int runTime = [testSuite getRuntime];
+    if (runTime == [MAX_RUNTIME_DISABLED intValue])
+        runTime = SECONDS_IN_HOUR;
+    NSString *time = NSLocalizedFormatString(@"Dashboard.Card.Seconds", [NSString stringWithFormat:@"%d", runTime]);
```

Comment:
It seems this code is duplicate, so maybe we can factor it?

Reply:
ok